app/views/contexts/summary/group.py

```diff
@@ -145,6 +144,7 @@ def serialize(self):
                 "title": self.title,
                 "blocks": self.blocks,
                 "links": self.links,
+                "empty_list_text": self.empty_list_text,
```

Comment:
I would call this `placeholder_text`, we just happen to use it for list at the moment.